### ISSUE TITLE
Questions about reproduce

### ISSUE THREAD

**Issue by xiaobaoxia** [fengyurenpingsheng/Learned-image-compression-with-GLLMM#1]
Hello, I want to reproduce the results of your MS-SSIM experiment, but the performance is always not good. It may be because there is an error in the loss function. If it is convenient, can you release the code of the training part?

**Comment by fengyurenpingsheng** [comment]
For MS-SSIM training,  you can use the loss function of PSNR to initialize, for example, train tens of thousands of rounds first, and then use the loss function of MS-SSIM for training. There is a library of MS-SSIM in tensorflow. But I am sorry that my current training code is not convenient to open source.

**Comment by fengyurenpingsheng** [comment]
> Hello, I want to reproduce the results of your MS-SSIM experiment, but the performance is always not good. It may be because there is an error in the loss function. If it is convenient, can you release the code of the training part?

For MS-SSIM training, you can use the loss function of PSNR to initialize, for example, train tens of thousands of rounds first, and then use the loss function of MS-SSIM for training. There is a library of MS-SSIM in tensorflow. But I am sorry that my current training code is not convenient to open source.

**Comment by xiaobaoxia** [comment]
Thank you very much for your reply, it has given me a lot of help. I will try this training strategy. ^_^

**Comment by xiaobaoxia** [comment]
Thank you very much for your reply, it has given me a lot of help. I will try this training strategy. ^_^
